### PR TITLE
fixed the help text for attach command

### PR DESCRIPTION
**Pull request by kensipe**

### CODE REVIEW

**Comment by jdef** [comment]
lgtm

File: cli/dcoscli/data/help/cluster.txt [hunk 1]
```diff
@@ -17,7 +17,7 @@ Usage:
 
 Commands:
     attach
-        List only the currently attached cluster.
+        Attaches the named cluster.
```
Comment (adamtheturtle) [inline cli/dcoscli/data/help/cluster.txt]: To be consistent with the other descriptions in this section, this should be "Attach the named cluster". Else the others should be "Lists CLI...", "Renames a cluster..." etc.

**Comment by kensipe** [comment]
@klueska rebased

**Comment by kensipe** [comment]
rebased

**Comment by kensipe** [comment]
rebased

**Comment by kensipe** [comment]
rebased

**Comment by kensipe** [comment]
@klueska @bamarni is there anything preventing a merge?

**Comment by bamarni** [comment]
@kensipe : nope, seems like a no-brainer to me ^^

**Pull request merged**